README.md

```diff
@@ -53,34 +53,25 @@ Every pull request should pass pre-commit stage to be merged
 
 ****For a quick start, please refer to our examples:****
 
-1. Exemplary usage of propagation with two methods - pure convolution and convolutional diffractive neural network.
-
-```commandline
-    python propagation_example.py
```

Comment:
For me it would be easier to run the examples directly from the file (press F5) instead of this.